packages/evolution-frontend/src/components/survey/GroupWidgets.tsx

```diff
@@ -32,16 +32,17 @@ import { CliUser } from 'chaire-lib-common/lib/services/user/userType';
 import { WidgetStatus } from '../../services/interviews/interview';
 import { GroupConfig } from 'evolution-common/lib/services/widgets/WidgetConfig';
 import DeleteGroupedObjectButton from './widgets/DeleteGroupedObjectButton';
+import { _isBlank } from 'chaire-lib-common/lib/utils/LodashExtensions';
 
 type GroupedObjectProps = InterviewUpdateCallbacks & {
     interview: UserInterviewAttributes;
     user: CliUser;
     path: string;
     label?: string;
-    section: string;
+    /** The name of the widget, to be used as key for translations */
     shortname: string;
```

Comment:
Do we still need shortname ?

Reply:
Yes we do, because it needs an identifier to get the default translation keys. But the survey developer does not need to define it himself. It is internal runtime data (it is the name of widget in the survey actually, which is not available in the widgetConfig)